pkg/cri/resource-manager/policy/builtin/memtier/dynamic_demotion.go

```diff
@@ -117,6 +114,28 @@ func copyPagePool(p pagePool) pagePool {
 	return c
 }
 
+func CreateDemoter(p *policy, pageMover PageMover) *demoter {
```

Comment:
Minor nit: most of go libraries are using convention for Object, there is NewObject function. Maybe here also NewDemoter?

Reply:
Ok, I'll update.